destination/destination.go

```diff
@@ -108,9 +111,13 @@ func (d *Destination) Configure(ctx context.Context, cfg map[string]string) erro
 	return nil
 }
 
+// registry registers StringObjectIDCodec.
+var registry = bson.NewRegistryBuilder().RegisterDefaultEncoder(reflect.String, writer.StringObjectIDCodec{}).Build()
```

Comment:
Let's move it up, above the `Writer` interface.

Reply:
And format it something like this:
```suggestion
var registry = bson.NewRegistryBuilder().
	RegisterDefaultEncoder(reflect.String, writer.StringObjectIDCodec{}).
	Build()
```

Reply:
fixed